vowpalwabbit/cb_explore_adf_squarecb.cc

```diff
@@ -351,7 +351,9 @@ base_learner* setup(VW::setup_base_i& stack_builder)
                .help("Upper bound on cost. Only used with --elim"))
       .add(make_option("cb_type", type_string)
                .keep()
-               .help("contextual bandit method to use in {ips,dr,mtr}. Default: mtr"));
+               .default_value("mtr")
+               .one_of({"ips", "dm", "dr", "mtr", "sm"})
```

Comment:
Similar to Jack's comment, in this case this reduction only works with mtr and if you scroll down in this file you can see it's handled. Not sure if the right approach is to only mention "mtr" here in one_of. John might have a stronger opinion. For the rest of the reductions, just make sure that after parsing you take a look at how this arg gets handled, to see if it was a case of wrong help message or correct.